backend/app/test/service/test_user_service.py

```diff
@@ -0,0 +1,148 @@
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_1
+        resp, status_code = user_service.create_user_follow_request({"id": 2})
+
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_confirm_user_follow_request(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+
+        resp, status_code = user_service.confirm_user_follow_request({"id": 1})
+
+        assert status_code == 201
+
+    @cleared_db
+    def test_get_a_user(self):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        db.session.add(user_1)
+        db.session.commit()
+
+        resp, status_code = user_service.get_a_user(1)
+
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_get_user_subscribers(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+        user_service.confirm_user_follow_request({"id": 1})
+
+        sub_list, status_code = user_service.get_user_subscribers()
+        assert status_code == 201
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_get_user_subscribers(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+        user_service.confirm_user_follow_request({"id": 1})
+
+        sub_list, status_code = user_service.get_user_subscribers()
+        assert status_code == 201
+
+    @cleared_db
+    @patch('app.main.service.user_service.get_current_user')
+    def test_get_user_subscribedto(self, mock):
+        user_1 =  User(fb_id='123', email=None, name='Jake')
+        user_2 = User(fb_id='123', email=None, name='Jane')
+        
+        user_1.followers.append(user_2)
+        db.session.add(user_1)
+        db.session.add(user_2)
+        db.session.commit()
+
+        mock.return_value = user_2
+        user_service.confirm_user_follow_request({"id": 1})
+
+        sub_list, status_code = user_service.get_user_subscribedto()
+
```

Comment:
For this test and the one above it we should test that "sublist" is equal to what we want.  For example in subscribedto we want the current user to be user_2.  That way we can confirm that sublist is equal to user_1.  This is the response object:
[
  {
    "name": "Jake",
    "user_id": 0
  }
]

Reply:
Yes, that's a good idea, will add